e2e/tests/timing/dom-content-loaded.test.ts

```diff
@@ -17,7 +17,7 @@ test(`should apply variants after DOMContentLoaded has fired`, skipIfIE(async t
 		.expect(log.length).eql(5)
 		.expect(log[0]).match(/^Delay/)
 		.expect(log[1]).match(/^DOMContentLoaded/)
-		.expect(log[2]).match(/^load/)
-		.expect(log[3]).match(/^Context applied/)
-		.expect(log[4]).match(/^Variant applied/);
+		.expect(log[2]).match(/^Context applied/)
```

Comment:
@mattstrom -- this is a difference between evaluation order locally vs browser stack isn't it? Do these tests have value?